src/Maker/MakeEntity.php

```diff
@@ -366,6 +366,17 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
         return $data;
     }
 
+    private function reOrderJsonArray($input)
+    {
+        // JSON must before JSON_ARRAY for autocomplete
+        $json_array = $input[Type::JSON_ARRAY];
+        unset($input[Type::JSON_ARRAY]);
+        // Put JSON_ARRAY at the end
+        $input[Type::JSON_ARRAY] = $json_array;
+
+        return $input;
+    }
```

Comment:
Hmm. Could we use https://www.php.net/manual/en/function.asort.php

The idea would be that, I think, `json` comes after `json_array` alphabetically... so that should sort it like we want it.

Reply:
`arsort` do the job, thanks @weaverryan